exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,14 @@
-const name = " Daniel  ";
-
-console.log(message);
+function lengthTrim(name) {
+  let trimeStr = name.trim();
+  return `my name is ${name} and my name is ${trimeStr.length} characters long`;
+}
+function lengthRegEx(name) {
+  let trimeStr = name.replace(/\s/g, "");
+  return `my name is ${name} and my name is ${trimeStr.length} characters long`;
+}
+const nametrim = " Rebwar a  ";
+const nameReqEx = " Rebwar a  ";
+const messagetrim = lengthTrim(nametrim);
+const messageRegEx = lengthRegEx(nameReqEx);
+console.log(`length with trim(): ${messagetrim}`);
+console.log(`length with RegExp: ${messageRegEx}`);
```

Comment:
Good use of trim and also regex, which has the advantage of getting rid of the space between words.